ios-sdk/TangramVC.swift

```diff
@@ -12,6 +12,7 @@ class TangramVC:  TGMapViewController{
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.queueSceneUpdate("sources.mapzen.url_params.api_key", withValue: "vector-tiles-7bfyfz8")
```

Comment:
Previously I was relying on the scene load that Tangram is does automatically if you don't specify a scene file; basically I have the `scene.yaml` placed in the correct spot by Xcode during the project copy. However that's bad form I'd imagine so I'll update to call explicitly from code.

As for the `applyUpdate()` method not being invoked - I don't see that exposed in the public header here - https://github.com/tangrams/tangram-es/blob/master/ios/src/TGMapViewController.h

Am I missing something?